Source/WebCore/platform/graphics/GraphicsContextState.cpp

```diff
@@ -100,6 +100,39 @@ void GraphicsContextState::mergeChanges(const GraphicsContextState& state, const
     }
 }
 
+void GraphicsContextState::mergeDiffs(const GraphicsContextState& state)
```

Comment:
The difference between "mergeChanges" and "mergeDiffs" is pretty subtle (diffs ... are changes). Also the fact that they take the same primary argument makes it even more subtle. Is there a better name for this?